README.md

```diff
@@ -14,6 +14,26 @@
 
 Download pre-compiled binaries for Mac, Windows and Linux from the [releases](https://github.com/dropbox/dbxcli/releases) page.
 
+### Mac OSX Installation of pre-compiled binaries
+1. If you don't already have it, create a `bin` directory under your home directory.
+```
+$ mkdir ~/bin
+```
+2. Add the following line at the end of your `~/.bash_profile` file.  [Link with instructions](https://natelandau.com/my-mac-osx-bash_profile/) on how to find this file
+```sh
+export PATH=$PATH:$HOME/bin
+```
+3. Download the `dbxcli` binary for OSX and rename it
+```sh
+$ cd ~/bin
+$ wget https://github.com/dropbox/dbxcli/releases/download/v2.0.9/dbxcli-darwin-amd64 
+$ mv dbxcli-darwin-amd64 dbxcli
+```
+4. Finally, Make the binary a Linux executable file and you are good to go!
```

Comment:
What's the "Linux" for (since this is in the Mac OS X section)? Did you mean to just say "Finally, make the binary executable ..."?

Reply:
haha, yep that is a typo.  Yes, you are right, should just say:
> Finally, make the binary executable...